src/utils/validators/location-vuelidate.js

```diff
@@ -0,0 +1,6 @@
+import { helpers } from 'vuelidate/lib/validators';
+
+export const zipValidator = helpers.regex(
+  'zip',
+  /^[0-9]{5}(?:-[0-9]{4})?$/,
```

Comment:
I don't think this will work for Canadian codes, right? We should have a more complete validator in the FE code you can steal.

Reply:
Please remove zipValidator and use only zipsValidator.